### PR TITLE
Overview page

### PR DESCRIPTION
**Pull request by royrutto**

### CODE REVIEW

File: src/cljs/hatti/views/dataview.cljs [hunk 1]
```diff
@@ -69,7 +72,7 @@
     (when (contains? (set views) view)
       (shared/transact-app-state! shared/app-state
                                   [:views :selected]
-                                  (fn [_] view))
+                                    (fn [_] view))
```
Comment (okal) [inline src/cljs/hatti/views/dataview.cljs]: Indentation.

File: src/cljs/hatti/views/dataview.cljs [hunk 2]
```diff
@@ -27,7 +30,7 @@
            :label "Summary Charts"
            :component chart-page}
    :details {:view :details
-             :label "Details"
+             :label "Settings"
```
Comment (okal) [inline src/cljs/hatti/views/dataview.cljs]: This no longer reflects the current state of things. `:component` value should probably be `settings-page`.

File: src/cljs/hatti/views/dataview.cljs [hunk 3]
```diff
@@ -95,8 +98,8 @@
                        (if (and (= view :map) no-geodata?)
                          [:a {:class "inactive" :title "No geodata"}
                           (name view)]
-                         [:a {:href (str "#/" (name view))
-                              :class (view->cls view)} label]))]
+                         [:a {:on-click (click-fn #(activate-view! view))
+                              :href "#" :class (view->cls view)} label]))]
```
Comment (okal) [inline src/cljs/hatti/views/dataview.cljs]: How does this play with client side routing? I think the reason the previous implementation did `(str "#/" (name view))` was to play well with Secretary, though I could be wrong about that.

File: src/cljs/hatti/views/overview.cljs [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+            [hatti.views :refer [overview-page]]))
+
+
+(defmethod overview-page :default
+  [{:keys [dataset-info]} owner]
+  "Om component for the overview page."
+  (om/component
+    (html
+      [:div.container
+       [:p "This is the default overview page holder."]])))
```
Comment (okal) [inline src/cljs/hatti/views/overview.cljs]: Is the `go` import being used here?

File: src/cljs/hatti/views/dataview.cljs (hunk 3 above)
Comment (prabhasp) [inline src/cljs/hatti/views/dataview.cljs]: Yup, we need to revert this change, since we want nav changes to reflect in the URL, and secretary to handle changes off of that.
